internal/impl/snowflake/streaming/parquet.go

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
```

Comment:
This is all enterprise. Did I mess that up?

Reply:
See the `userdata_converter` stuff. Also, the `int128` module, but I guess that's OK since it's a separate module.

Reply:
I think I got the license right for `userdata_converter`?

Yeah the licensing is a little strange with `int128` since it's a bunch of different libraries mashed together. Hopefully I can upstream the arrow stuff and then we can drop it